javascript/packages/orchestrator/src/chain-spec.ts

```diff
@@ -13,6 +13,8 @@ import { ChainSpec, HrmpChannelsConfig, Node } from "./types";
 const JSONbig = require("json-bigint")({ useNativeBigInt: true });
 const debug = require("debug")("zombie::chain-spec");
 
+const JSONStream = require("JSONStream");
```

Comment:
For consistency reasons lets use import here:

```suggestion
import JSONStream  from "JSONStream";
```